lms/static/js/staff_debug_actions.js

```diff
@@ -86,26 +83,44 @@ var StaffDebug = (function() {
         });
     };
 
-    rescore = function(locname, location) {
-        this.do_idash_action({
+    var rescore = function(locname, location) {
+        this.doInstructorDashAction({
             locationName: locname,
             location: location,
             method: 'rescore_problem',
             success_msg: gettext('Successfully rescored problem for user {user}'),
             error_msg: gettext('Failed to rescore problem.'),
-            delete_module: false
+            only_if_higher: false
+        });
+    };
+
+    var rescoreIfHigher = function(locname, location) {
+        this.doInstructorDashAction({
+            locationName: locname,
+            location: location,
+            method: 'rescore_problem',
+            success_msg: gettext('Successfully rescored problem (if higher) for user {user}'),
+            error_msg: gettext('Failed to rescore problem (if higher).'),
```

Comment:
@catong Here are 2 status messages that were added for the "if higher" case.  Any suggestions?